src/core/ArxGame.cpp

```diff
@@ -1294,7 +1294,17 @@ void ArxGame::updateFirstPersonCamera() {
 				g_playerCamera.m_pos.x = player.pos.x + vect.x;
 				g_playerCamera.m_pos.z = player.pos.z + vect.z;
 			}
-			
+
+			// ----------------
+
+			int distance = 200.0f;
```

Comment:
Curious why you're declaring distance as an int. Was this perhaps a mistake?

Reply:
good catch, it started out as an int, but later I've changed it to a float. will update it later.